client/modules/User/components/ExamplePreview.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { PropTypes } from 'prop-types';
+import axios from 'axios';
+import { createGlobalStyle } from 'styled-components';
+import getConfig from '../../../utils/getConfig';
+import EmbedFrame from '../../Preview/EmbedFrame';
+import { registerFrame } from '../../../utils/dispatcher';
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
+  }
+`;
+
+function ExamplePreview({ url }) {
+  const [sketch, setSketch] = useState(null);
+  registerFrame(window.parent, getConfig('EDITOR_URL'));
+
+  useEffect(() => {
+    async function getRandomProject() {
+      // Find example projects
+      const response = await axios.get(url);
+      const projects = response.data;
+      if (!projects) return null;
+      setSketch(projects);
+      console.log('sketch', sketch);
+      return projects;
+    }
+    getRandomProject();
+  }, [url]);
```

Comment:
In the 404 page this part of the code gets all sketches from the p5 user and returns a random one.  Here we want to get the one specific sketch by its id.
https://github.com/processing/p5.js-web-editor/blob/41529a3bb401e6516ef2d62d9fca9ba8aa21bf43/client/modules/IDE/actions/project.js#L57-L58